massgov/mayflower
# DP-23521 Front end enhancement for Breadcrumbs & Offered By, medium & small

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Places breadcrumbs and relationship indicators for mobile.
## Related Issue / Ticket

- [JIRA issue](https://massgov.atlassian.net/browse/DP-23521)

## Steps to Test


1. This could either be merged into the integration branch and tested in feature3 or integrated into another feature environment.
2. Open the environment at less than 910px and compare with the mockup (make sure to refresh or to test on an actual mobile phone)

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.

<img width="607" alt="j-r-capture 2021-12-07 a la(s) 16 26 10" src="https://user-images.githubusercontent.com/141685/145093305-be69dffd-8d28-4ee8-a226-544cd358b832.png">



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
